src/__tests__/demo.js

```diff
@@ -59,30 +59,30 @@ window.adobeDataLayer.push({
   }
 });
 
-window.adobeDataLayer.getState().component.carousel.carousel1;
+// window.adobeDataLayer.getState().component.carousel.carousel1;
```

Comment:
I'd change it to:
console.log(window.adobeDataLayer.getState('component.carousel.carousel1'))